src/clients/AcrossConfigStoreClient/AcrossConfigStoreClient.ts

```diff
@@ -76,7 +76,7 @@ export class AcrossConfigStoreClient extends BaseAbstractClient {
 
   protected rateModelDictionary: across.rateModel.RateModelDictionary;
   public firstBlockToSearch: number;
-  public latestBlockNumber = 0;
+  public latestBlockSearched = 0;
```

Comment:
Should we put this in `BaseAbstractClient.ts` ? This generic update functionality feels like it could belong there and would give greater consistency across all of the inheriting classes.

Reply:
you mean move these generic class properties like this? 7a07154d3f828583ef3c7a0dd577fa3cc474a48f

Reply:
Yeah - the batched update feature is basically common between all of the classes, but I have noticed that we have some slight differences between them (and duplicated code, to some extent). We can improve on that by pulling it out to the common class that they all inherit from :+1:.